docs/content/en/docs/references/cli/_index.md

```diff
@@ -811,6 +812,21 @@ Usage:
 Use "skaffold options" for a list of global command-line options (applies to all commands).
 
 
+```
+
+### skaffold survey
+
+Show Skaffold survey url
+
+```
+
+
+Usage:
+  skaffold survey [options]
+
+Use "skaffold options" for a list of global command-line options (applies to all commands).
```

Comment:
Are there really other options that apply to `skaffold survey`? :-)

Reply:
haha, good point...and does `--filename` need to be a supported flag?

Reply:
unfortunately, i cant find an easy way to hide this help text. Please feel free to fix this in a follow up PR later.